schema/transportation/segment.yaml

```diff
@@ -258,17 +257,18 @@ properties:
                        - gatedKey
                      default:
                        enum: [legal]
-                   direction:
-                     description: direction of travel on the source that pertains to the relation
-                     "$ref": "#/$defs/propertyDefinitions/direction"
-                   via:
+                   sequence:
                      description: Sequence of ordered segements that needs to be traversed in order for the relation to be effective
                      type: array
                      items:
-                       description: GERS ID of the connected segments
+                       description: GERS ID of the connected segments and the relevent connector
```

Comment:
nit: the last vowel in relev*a*nt is an *a* not an *e*.

----

I think that where we're landing on relationships is that this ID will not be the GERS ID, it will be the "technical" or "feature" ID.

I would be tempted to just phrase this as:

> Pair of IDs for the segment being traversed and the connector used to access that segment

Reply:
Fixed